packages/core/src/app/address/googleAutocomplete/GoogleAutocompleteFormField.tsx

```diff
@@ -53,7 +53,7 @@ const GoogleAutocompleteFormField: FunctionComponent<GoogleAutocompleteFormField
             id: getAddressFormFieldInputId(name),
             'aria-labelledby': labelId,
             placeholder: useFloatingLabel ? ' ' : placeholder,
-            labelText: useFloatingLabel ? labelContent : null,
+            'data-labeltext': useFloatingLabel ? labelContent : null,
```

Comment:
This is to avoid a React warning for passing `labelText` as an invalid input field prop.